GoogleForCreators/web-stories-wp
# UI Simplification: Text Panel

## Summary
Updates Text Panel based on the new UI.


## Relevant Technical Choices



## To-do

- [x] Add eyedropper to the color input



## User-facing changes

| Before | After |
|--------|------|
| <img width="304" alt="Screenshot 2021-10-29 at 17 21 00" src="https://user-images.githubusercontent.com/3294597/139451616-cc1dae93-dd51-4785-9540-960595bacf6a.png"> | <img width="305" alt="Screenshot 2021-10-29 at 17 19 01" src="https://user-images.githubusercontent.com/3294597/139451467-e5e764d5-6d2d-446c-ab4c-72fb3c287f0e.png"> |


## Testing Instructions






This PR can be tested by following these steps:

1. Add a text element
2. Verify the Text Box panel is now part of the Text panel.
3. Verify the inputs and generally the style looks as in [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=10250%3A435667)
4. Verify that when the panel is open, the Text Magic button is visible in the Panel header. Verify it is hidden when collapsing the Text Panel.
Note: Ignore the Recent Saved Styles section, this will be done separately.


### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #9462
